docs/sources/mimir/release-notes/v2.12.md

```diff
@@ -0,0 +1,122 @@
+---
+title: "Grafana Mimir version 2.12 release notes"
+menuTitle: "V2.12 release notes"
+description: "Release notes for Grafana Mimir version 2.12"
+weight: 1000
+---
+
+# Grafana Mimir version 2.12 release notes
+
+Grafana Labs is excited to announce version 2.12 of Grafana Mimir.
+
+The highlights that follow include the top features, enhancements, and bugfixes in this release. For the complete list of changes, see the [changelog](https://github.com/grafana/mimir/blob/main/CHANGELOG.md).
+
+## Features and enhancements
+
+- **Added mimir-distroless docker image, built upon the distroless. This improvement is geared towards minimising attack surfaces and mitigating potential CVEs by trimming down the dependencies within the image. After comprehensive testing, we plan to shift from the default docker image to distroless.
```

Comment:
Let's include the name of docker image that people can try for themselves.

Reply:
@ying-jeanne what is the name of these docker images?

Reply:
`mimir-distroless` is the name of the image. (https://hub.docker.com/r/grafana/mimir-distroless/tags)